Transbank/PatpassRest/PatpassComercio/Inscription.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using Newtonsoft.Json;
+using Transbank.Common;
+using Transbank.Exceptions;
+using Transbank.Patpass.Common;
+using Transbank.Patpass.PatpassComercio.Requests;
+using Transbank.Patpass.PatpassComercio.Responses;
+
+namespace Transbank.Patpass.PatpassComercio
+{
+    public static class Inscription
+    {
+        private static string _commerceCode = "";
```

Comment:
You are missing the default CommerceCode and ApiKey

Reply:
already fixed, I've miss the push 15d8e7d